### ISSUE TITLE
Please add  autoEscape={true} in README

### ISSUE THREAD

**Issue by StanislavMayorov** [clauderic/react-native-highlight-words#17]
Hi. Thank you for the package.
Please add  `autoEscape={true}` in README Usage example. I think it will be useful.

Why is `autoEscape` default value false?

**Comment by CyxouD** [comment]
>Why is autoEscape default value false?

Agree, it would help to avoid also this issue https://github.com/clauderic/react-native-highlight-words/issues/16 in some cases :)
